.github/workflows/main.yml

```diff
@@ -0,0 +1,36 @@
+name: Tests
+on: [pull_request, push]
```

Comment:
I'm unaware If this has been discussed, but should the CI run on all branches, or specify only on main branch?

Reply:
Good catch, Mia see https://github.com/canonical/operator-template/blob/d54942b30e9f1f0b2eb402d876f41bfd3a9efa62/.github/workflows/ci.yaml#L5